lib/lotus/helpers/html_helper/html_builder.rb

```diff
@@ -232,6 +233,33 @@ def tag(name, content = nil, attributes = nil, &blk)
           self
         end
 
+        # Define a HTML fragment
+        #
+        # @param content [String,Lotus::Helpers::HtmlHelper::HtmlBuilder,NilClass] the optional content
+        # @param blk [Proc] the optional nested content espressed as a block
+        #
+        # @return [self]
+        #
+        # @since 0.1.0
+        # @api public
+        #
+        # @see Lotus::Helpers::HtmlHelper
+        #
+        # @example
+        #   html.fragment('Lotus') # => Lotus
+        #
+        #   html do
+        #     p 'hello'
+        #     p 'lotus'
+        #   end
+        #   # =>
+        #     <p>hello</p>
+        #     <p>lotus</p>
+        def fragment(content = nil, &blk)
+          @nodes << HtmlFragment.new(blk || content)
```

Comment:
@huynhquancam Why this fragment concept is needed at all? If we pass a block to `#initialize` we can `instance_eval` it.

I don't like the idea of `html.fragment('Lotus')`, it exposes an internal concept. The expected API is a mapping for HTML tags. For instance, to expose `#div` and `#span` for `<div>` and `<span>`, respectively.

If we still need this method and `HtmlFragment`, it should only accept `&blk` and not `content` as well.